knative-operator/pkg/webhook/knativeserving/webhook_mutating.go

```diff
@@ -26,7 +26,7 @@ func MutatingWebhook(mgr manager.Manager) (webhook.Webhook, error) {
 	return builder.NewWebhookBuilder().
 		Name("mutating.knativeserving.openshift.io").
 		Mutating().
-		Operations(admissionregistrationv1beta1.Create).
+		Operations(admissionregistrationv1beta1.Create, admissionregistrationv1beta1.Update).
```

Comment:
Are you sure this is needed? I would expect the controller to handle this.

Reply:
And the controller even does, but it leads to weird behavior when you override the entirety of KnativeServing with an update. The result you get back is not what's reflected on the cluster.